packages/contracts-bedrock/test/Predeploys.t.sol

```diff
@@ -71,4 +75,86 @@ contract PredeploysTest is CommonTest {
+        );
+    }
+
+    function test_preinstall_safeSingletonFactory_succeeds() external {
+        assertEq(
+            Predeploys.SafeSingletonFactory.code,
+            hex"7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+        );
+    }
+
+    function test_preinstall_deterministicDeploymentProxy_succeeds() external {
+        assertEq(
+            Predeploys.DeterministicDeploymentProxy.code,
+            hex"7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf3"
+        );
+    }
+
+    function test_preinstall_multisendv130_succeeds() external {
+        assertEq(
+            Predeploys.MultiSend_v130.code,
+            hex"60806040526004361061001e5760003560e01c80638d80ff0a14610023575b600080fd5b6100dc6004803603602081101561003957600080fd5b810190808035906020019064010000000081111561005657600080fd5b82018360208201111561006857600080fd5b8035906020019184600183028401116401000000008311171561008a57600080fd5b91908080601f016020809104026020016040519081016040528093929190818152602001838380828437600081840152601f19601f8201169050808301925050505050505091929192905050506100de565b005b7f000000000000000000000000998739bfdaadde7c933b942a68053933098f9eda73ffffffffffffffffffffffffffffffffffffffff163073ffffffffffffffffffffffffffffffffffffffff161415610183576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004018080602001828103825260308152602001806102106030913960400191505060405180910390fd5b805160205b8181101561020a578083015160f81c6001820184015160601c6015830185015160358401860151605585018701600085600081146101cd57600181146101dd576101e8565b6000808585888a5af191506101e8565b6000808585895af491505b5060008114156101f757600080fd5b8260550187019650505050505050610188565b50505056fe4d756c746953656e642073686f756c64206f6e6c792062652063616c6c6564207669612064656c656761746563616c6ca26469706673582212205c784303626eec02b71940b551976170b500a8a36cc5adcbeb2c19751a76d05464736f6c63430007060033"
+        );
+    }
+
+    /// @dev The domain separator commits to the chainid of the chain
+    function test_preinstall_permit2_succeeds() external {
+        bytes32 domainSeparator = IEIP712(Predeploys.Permit2).DOMAIN_SEPARATOR();
+        bytes32 typeHash =
+            keccak256(abi.encodePacked("EIP712Domain(string name,uint256 chainId,address verifyingContract)"));
+        bytes32 nameHash = keccak256(abi.encodePacked("Permit2"));
+        uint256 chainId = block.chainid;
+        bytes memory encoded = abi.encode(typeHash, nameHash, chainId, Predeploys.Permit2);
+        bytes32 expectedDomainSeparator = keccak256(encoded);
+        assertEq(domainSeparator, expectedDomainSeparator, "Domain separator mismatch");
+    }
```

Comment:
nit: Would you mind moving this test above code assertions (`test_preinstall_multicall3_succeeds` test looks to be the first one) and put a divider so it's clear where code assertion tests should go vs other tests? Just makes it easier to find this and other tests given large hex strings

ie
```
    function test_preinstall_permit2_succeeds() external { ... }

    //////////////////////////////////////////////////////
    /// Code Assertion Tests
    //////////////////////////////////////////////////////

    function test_preinstall_multicall3_succeeds() external { ... }
```

Reply:
i have addressed the comment :D